patches/common/chromium/fix_disable_usage_of_pthread_fchdir_np_and_pthread_chdir_np_in_mas.patch

```diff
@@ -0,0 +1,54 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@slack-corp.com>
+Date: Mon, 4 Mar 2019 14:46:48 -0800
+Subject: fix: disable usage of pthread_fchdir_np and pthread_chdir_np in MAS
+ builds
```

Comment:
hrrrrm is this what `export-patches` produced? the 2-line subject line is probably gonna cause issues

Reply:
Yup, this is exactly what the script produced.  It looks like the builds are OK and handle it though?

Reply:
hmmm... does it survive `import-patches && export-patches`? if not, might have to kick the `export-patches` script